js/common.js

```diff
@@ -185,6 +185,12 @@ function getCheckboxValue(field){
 	}
 }
 
+function validateEmail(email) {

```

Comment:
We're using this function in two places in https://github.com/TAMULib/CORAL-Resources/blob/issues_feature_remastered/js/resource.js  In the first, it's in a non-submitting context, so HTML5 validation isn't useful (L.187).

In the other (L.161), it's as part of the usual Coral way of pushing error messages to nearby spans to notify users of what they need to fix. If we rely on browser validation for this one part, it means the user has to look in more than one place and for more than one style of error feedback. 

The alternative is refactoring the error checking to rely on the JS validation API and HTML5 'required', but no other validation in Coral uses this that I'm aware of, so that seems more like a module wide feature beyond the scope of the Issues/Downtime feature.

Also, the actual email validation regex is the same and in some cases, stricter, than what the browsers themselves are using.